tests/src-android/cgeo/geocaching/connector/gc/GCConnectorTest.java

```diff
@@ -36,14 +36,14 @@ public static void testGetViewport() {
                 final Viewport viewport = new Viewport(new Geopoint("N 52° 25.369 E 9° 35.499"), new Geopoint("N 52° 25.600 E 9° 36.200"));
                 final SearchResult searchResult = ConnectorFactory.searchByViewport(viewport, tokens);
                 assertThat(searchResult).isNotNull();
-                assertThat(searchResult.isEmpty()).isFalse();
-                assertThat(searchResult.getGeocodes()).contains("GC4ER5H");
+                assertThat(searchResult.getGeocodes()).doesNotContain("GC4ER5H");
             }
 
             {
                 final Viewport viewport = new Viewport(new Geopoint("N 52° 24.000 E 9° 34.500"), new Geopoint("N 52° 26.000 E 9° 38.500"));
                 final SearchResult searchResult = ConnectorFactory.searchByViewport(viewport, tokens);
                 assertThat(searchResult).isNotNull();
+                assertThat(searchResult.isEmpty()).isFalse();
```

Comment:
See comment above. This is not explicitly needed here but on the first test IMHO.